runtime/doc/eval.txt

```diff
@@ -2476,6 +2477,20 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		If {command} is a |String|, it returns {command} output.
+		If {command} is a |List|, it returns concatenated all outputs
+		of each items.
```

Comment:
“All” should not be there because there already is “each”. “Item” should be singular because it relates to each: http://www.dictionary.com/browse/each:

> The adjective each is always followed by a singular noun: each person; each book.